eng/Version.Details.xml

```diff
@@ -298,11 +298,11 @@
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>72b7d236ad634c2280c73499ebfc2b594995ec06</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.App.Internal" Version="6.0.0-alpha.1.20560.10">
+    <Dependency Name="Microsoft.NETCore.BrowserDebugHost.Transport" Version="6.0.0-alpha.1.20560.10">
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>72b7d236ad634c2280c73499ebfc2b594995ec06</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.BrowserDebugHost.Transport" Version="6.0.0-alpha.1.20560.10">
+    <Dependency Name="VS.Redist.Common.NetCore.SharedFramework.x64.6.0" Version="6.0.0-alpha.1.20560.10">
```

Comment:
When will Microsoft.NETCore.BrowserDebugHost.Transport and VS.Redist.Common.NetCore.SharedFramework.x64.6.0 have different versions❔ Using both does not seem useful if both are non-shipping packages and we don't need VS.Redist.Common.NetCore.SharedFramework.x64.6.0 for anything other than its version.

Reply:
They'll always have the same version. It felt more correct to have use the VS insertion package when it related to installers (where the build references the NETCoreApp installers) and the Browser Debug Host when relating to other usages. I'd be fine just using the Browser Debug Host version everywhere if you'd prefer.

Reply:
Yup, that's my preference😺